packages/design-picker/package.json

```diff
@@ -49,7 +49,8 @@
 		"@wordpress/element": "^2.19.1",
 		"@wordpress/i18n": "^3.19.0",
 		"react": "^16.12.0",
-		"react-dom": "^16.12.0"
+		"react-dom": "^16.12.0",
+		"debug": "^4.1.1"
```

Comment:
Do we need to keep this?

Reply:
I was wondering the same.  debug things wouldn't go into production right?

Reply:
I'm not sure, to be honest. `yarn` does some clever things with the namespaces, but the linter complains. Peer dependency seemed about right to me.

I could pull the debug out, but you'll be grateful for them if you need to work on this code in the future ;)

@autumnfjeld: Apparently they do!  I thought there was some magic to eliminate the debug calls in release, but `localStorage.setItem('debug', '*')` dumps a ton of output even logged out and unproxied.

Reply:
Good to know!!!!  TIL 👍😍

And you are right, we'll be grateful for debugging logging.